### ISSUE TITLE
Bad flux unit in FITS output?

### ISSUE THREAD

**Issue by allegroLeiden** [lime-rt/lime#61]
LIME calculates output flux densities in Jy/pixel. The CASA spectrum extractor (CASA 4.3.1) does not recognize this as a proper unit. Maybe we should convert to Jy/arcsec^2, which is recognized?

**Comment by smaret** [comment]
Does it recognize Kelvins?

**Comment by smaret** [comment]
I'm not in favor of converting the Jy/pixel to Jy/str. This may confuse users that expect to get Jy/pixel if the `img[i]->unit` parameter is set to 1 (as in [previous LIME versions](http://lime.readthedocs.org/en/latest/usermanual.html#images)). What *could* be done is to add an additional unit code (5?) for Jy/str.

**Comment by imckstewart** [comment]
No it doesn't like Kelvin, it wants a beam width to be defined for that.

**Comment by imckstewart** [comment]
I agree about adding extra unit codes rather than redefining old ones.

**Comment by smaret** [comment]
> No it doesn't like Kelvin, it wants a beam width to be defined for that.

Weird. Kelvins units are homogeneous to Jy/str. They do not depend on the beam width -- unless you're talking about antenna or mean beam temperature rather than *brightness* temperature.

> I agree about adding extra unit codes rather than redefining old ones.

OK, I'm labelling this as "enhancement' then.

**Comment by tlunttil** [comment]
I'm continuing my crusade against dangling issues... There's already unit W m^-2 Hz^-1 sr^-1, which is just Jy/sr times 10^26. Is there any need for separate unit with Janskies? Or maybe change unit==2 from SI to Janskies? Astronomers using SI units are a rare breed. That would break backward compatibility, though. 

Additionally, Janskies (or something else) per square arcsec could be a useful unit.
